addons/mail/models/mail_channel.py

```diff
@@ -1204,7 +1204,7 @@ def execute_command_help(self, **kwargs):
         else:
             all_channel_partners = self.env['mail.channel.partner'].with_context(active_test=False)
             channel_partners = all_channel_partners.search([('partner_id', '!=', partner.id), ('channel_id', '=', self.id)])
-            msg = _("You are in a private conversation with <b>@%s</b>.", html_escape(channel_partners[0].partner_id.name if channel_partners else _('Anonymous')))
+            msg = _("You are in a private conversation with <b>@%s</b>.", html_escape(_(" @").join(_(partners.partner_id.name) for partners in channel_partners) if channel_partners else _('Anonymous')))
```

Comment:
Hello, thank you for the PR.

There should be no translation around this part `_(partners.partner_id.name)`.

Also please add a test to cover the fix.

(And not that I care much but I believe the same issue is already there in v14 too if we want to fix everything.)

And actually, while we are at it, it should not actually say `with public user` but `with *whatever is the anonymous name*` in case of livechat.
But correctly naming the livechat user could be considered another fix than this PR which aims to list missing users.

Reply:
@sea-odoo Thank you. I will edit as requested

Reply:
> @sea-odoo Thank you. I will edit as requested

@seb-odoo :-D

Reply:
@seb-odoo I fixed this PR in version 14. Can you review it for me?
https://github.com/odoo/odoo/pull/103224/files#diff-33d42486197097b74dd923bd2e89a94d7a19c82c31c981d51e22411c0360e584L1123-R1124